src/os/CMakeLists.txt

```diff
@@ -86,7 +86,13 @@ if(HAVE_LIBZFS)
 endif()
 
 if(WITH_SPDK)
-  target_link_libraries(os
+   set(DPDK_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/src/spdk/dpdk/build/include)
+   set(check_LIBRARIES ${CMAKE_SOURCE_DIR}/src/spdk/dpdk/build/lib/librte_eal.a
```

Comment:
no need to duplicate the variables in `Finddpdk.cmake` here.

Reply:
i meant we don't need to use `check_LIBRARIES` which we don't **check** anymore.

and please move these `set()` statements into `Builddpdk.cmake`.

Reply:
@liupan1111 this is not addressed.